share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -132,12 +148,31 @@ function(rocm_setup_version)
         if(PARSE_VERSION MATCHES "^[0-9]+\\.[0-9]+$")
             set(PARSE_VERSION ${PARSE_VERSION}.0)
         endif()
-        if(PARSE_NO_GIT_TAG_VERSION)
-            set(PACKAGE_VERSION ${PARSE_VERSION})
-        else()
+        if(PARSE_ADD_GIT_TAG_VERSION)
```

Comment:
This should be enabled by default. We always want the commit count, hash and build id in the version.

Reply:
Changes are done, based on the new scheme which was discussed.
An example
git hub build/local machine build - rocm-cmake_3.6.0-<commitHash>_amd64.deb
​CI non release build - rocm-cmake_3.6.0.30800-dnnhc.3049_amd64.deb
​CI release build - rocm-cmake_3.6.0.30800-1_amd64.deb

Reply:
What is an example name for a rocBLAS package with rocblas shipping with semantic version 1.1 and rocm version 3.8.0?

Reply:
git hub build/local machine build - 
rocblas_1.1.22-gitCommitHash_amd64.deb, rocblas-1.1.22-gitCommitHash.el7.x86_64.rpm
​
CI non release build - 
rocblas_1.1.22.30800-dnnhc.3049_amd64.deb, rocblas-1.1.22.30800-dnnhc.3049.el7.x86_64.rpm

​CI release build - 
rocblas_1.1.22.30800-12_amd64.deb, rocblas-1.1.22.30800-12.el7.x86_64.rpm

Reply:
Yes, 12 is the build ID from Jenkins on the release job.
Similar 3049 is the build ID from Jenkins from the mainline build and dnnhc stands for - "dkms-no-npi-hipclang" job